### ISSUE TITLE
TypeError: _TypedDictMeta.__new__() got an unexpected keyword argument 'closed'

### ISSUE THREAD

**Issue by ddobrow** [vega/altair#3709]
### What happened?

```python
TypeError                                 Traceback (most recent call last)
Cell In[2], line 2
    1 # Import altair with an abbreviated alias
----> 2 import altair as alt
    4 # Load a sample dataset as a pandas DataFrame
    5 from vega_datasets import data

File ~\Miniconda3\envs\altair\Lib\site-packages\altair\__init__.py, line 650
    646 def __dir__():
    647     return __all__
--> 650 from altair.vegalite import *
    651 from altair.vegalite.v5.schema.core import Dict
    652 from altair.jupyter import JupyterChart

File ~\Miniconda3\envs\altair\Lib\site-packages\altair\vegalite\__init__.py, line 2
----> 2 from .v5 import *

File ~\Miniconda3\envs\altair\Lib\site-packages\altair\vegalite\v5\__init__.py, line 4
----> 4 from .api import *

File ~\Miniconda3\envs\altair\Lib\site-packages\altair\vegalite\v5\api.py, line 618
    618 class _ConditionClosed(TypedDict, closed=True, total=False):  # type: ignore[call-arg]
TypeError: _TypedDictMeta.__new__() got an unexpected keyword argument 'closed'
```

### Which version of Altair are you using?

5.4.1

**Comment by mattijn** [comment]
Thanks for raising! 

The `closed` argument within a TypedDict is part of [PEP 728](https://peps.python.org/pep-0728/) and not available on earlier Python versions.

This was potentially fixed by https://github.com/vega/altair/pull/3683, which was included in the latest released version (v5.5.0). 
Double checking with @dangotbanned, that is right what I'm saying?

**Comment by dangotbanned** [comment]
> This was potentially fixed by [#3683](https://github.com/vega/altair/pull/3683), which was included in the latest released version (v5.5.0).
> Double checking with [<img alt="" width="16" height="16" src="https://avatars.githubusercontent.com/u/125183946?s=64&amp;v=4">@dangotbanned](https://github.com/dangotbanned?rgh-link-date=2024-12-11T11%3A55%3A37.000Z), that is right what I'm saying?

100% right @mattijn, thanks for checking in.
Should be fixed by updating to https://github.com/vega/altair/releases/tag/v5.5.0
